<launch>
	<node pkg="tanwaylidar_view" name="TanwayLidar_MainNode" type="TanwayLidar_MainNode" output = "screen">
		<param name="frame_id" type="string" value="TanwayTP" />
		<param name="topic" type="string" value="/tanwaylidar_pointcloud" />
		<param name="LidarType" type="int" value="6"/> 
		
		<!--本机IP地址-->
		<param name="LocalHost" type="string" value="192.168.111.204" />
		<!--本机接收雷达数据的端口号-->
		<param name="LocalPort" type="int" value="5600" />
		<!--l雷达设备的IP地址-->
		<param name="LidarHost" type="string" value="192.168.111.51" />
		<!--雷达设备发送端的端口号-->
		<param name="LidarPort" type="int" value="5050" />
		<!--左偏移角度[15-45]-->
		<param name="LeftMoveAngle" type="double" value="30.0" />		
		<!--右偏移角度[15-45]--> 
		<param name="RightMoveAngle" type="double" value="30.0" />	
	</node>

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find tanwaylidar_view)/rviz/tanwayrun.rviz" />
</launch>
